modules/virt-cnv-virt-alerts.adoc

```diff
@@ -0,0 +1,487 @@
++
+[source,terminal]
+----
+oc get deployment -n $NAMESPACE virt-controller -o yaml
+----
+
+. Check the manager pod's logs to determine why creating the virt-controller pods fails.
++
+[source,terminal]
+----
+oc get logs <virt-controller-pod>
+----
+
+[NOTE]
+====
+Do not use a name for the virt-controller pod that is similar to virt-controller-7888c64d66-dzc9p.
+====
+
+There may be several pods that run virt-controller.
+
+.Resolution
+
+There are several reasons why detection of no running virt-controller occurs. Identify the root cause and take appropriate action.
+
+* Node resource exhaustion
+* Not enough memory on the cluster
+* Nodes are down
+* The API server overloads, such as when the scheduler is not 100% available)
+* Networking issues
+
+Otherwise, open a support issue and provide the information gathered in the troubleshooting process.
+
+//VirtControllerRESTErrorsBurst Alert
+[id="VirtControllerRESTErrorsBurst_{context}"]
+== VirtControllerRESTErrorsBurst alert
+
+.Description
+
+More than 80% of the REST calls failed in virt-controller in the last five minutes.
+
+.Reason
+
+Virt-controller has potentially fully lost connectivity to the API server. This loss does not affect running workloads, but propagation of status updates and actions like migrations cannot occur.
+
+.Troubleshoot
+
+There are two common error types associated with virt-controller REST call failure:
+
+* The API server overloads, causing timeouts. Check the API server metrics and details like response times and overall calls.
+
+* The virt-controller pod cannot reach the API server. Common causes are:
+** DNS issues on the node
+** Networking connectivity issues
+
+.Resolution
+
+Check the virt-controller logs to determine if it cannot connect to the API server at all. If so, delete the pod to force a restart. The issue normally relates to DNS or CNI issues outside of the scope of this alert.
+
+//VirtHandlerRESTErrorsBurst Alert
+[id="VirtHandlerRESTErrorsBurst_{context}"]
+== VirtHandlerRESTErrorsBurst alert
+
+.Description
+
+More than 80% of the REST calls failed in virt-handler in the last five minutes.
+
+.Reason
+
+Virt-handler lost the connection to the API server. Running workloads on the affected node still run, but status updates cannot propagate and actions such as migrations cannot occur.
+
+.Troubleshoot
+
+There are two common error types associated with virt-operator REST call failure:
+
+* The API server overloads, causing timeouts. Check the API server metrics and details like response times and overall calls.
+
+* The virt-operator pod cannot reach the API server. Common causes are:
+** DNS issues on the node
+** Networking connectivity issues
+
+.Resolution
+
+If the virt-handler cannot connect to the API server, delete the pod to force a restart. The issue normally relates to DNS or CNI issues outside of the scope of this alert. Identify the root cause and take appropriate action.
+
+//VirtOperatorDown Alert
```

Comment:
Same comment as above.

Reply:
This **Troubleshooting** and **Resolution** for this alert should be the similar to alert: VirtControllerDown

just change virt-controller to virt-operator